### PR TITLE
added joke feature

### PR DESCRIPTION
**Pull request by rajiv8**
## Added joke functionality..

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
 import discord
+from urllib import request
+import json
 
 # this is our bot 
 bot = discord.Client()
 
+# api request for joke 
+def getJoke():
+    url = request.urlopen("https://official-joke-api.appspot.com/jokes/programming/random")
```
Comment (KamalDGRT) [inline app.py]: Lets have the URL in the `.env` file.

File: app.py [hunk 2]
```diff
@@ -1,10 +1,29 @@
 # import discord package 
 # pip install discord
 import discord
+from urllib import request
+import json
```
Comment (KamalDGRT) [inline app.py]: We usually import the packages in the ascending order.

**Review by KamalDGRT — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (rajiv8) [reply app.py]: DONE , Please Check .

**Comment by KamalDGRT** [comment]
Closing this PR as we are going with a new file structure.

**Pull request closed without merging**